renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1449,6 +1449,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
   {
     if(fmt.compByteWidth == 4)
       ret = DXGI_FORMAT_R32_TYPELESS;
+    else if(fmt.compByteWidth == 3)
+      ret = DXGI_FORMAT_R24_UNORM_X8_TYPELESS;
```

Comment:
I have added this for the case of D24 formats and it's working properly for me. I am not sure what would happen for D24S8, I was under the impression that those are considered "special" and would be handled by the switch statement in the beginning of the function.